subprojects/language-java/src/integTest/groovy/org/gradle/javadoc/JavadocIntegrationTest.groovy

```diff
@@ -61,6 +61,20 @@ class JavadocIntegrationTest extends AbstractIntegrationSpec {
         file("build/docs/javadoc/Foo.html").text.contains("""Hey Joe!""")
     }
 
+    @Issue("https://github.com/gradle/gradle/issues/1090")
+    def "escape single quotes in javadoc options"() {
+        buildFile << """
+            apply plugin: "java"
+            javadoc.options.header = "where you goin' with that gun in your hand"
+        """
+
+        file("src/main/java/Foo.java") << "public class Foo {}"
+
+        when: run("javadoc", "-i")
+        then:
```

Comment:
Thank you, will fix all the issues you noted.

Regarding `succeeds` rather than `run` and a new line between `when` and `then`: Several other tests in that file are inconsistent and I will change them if desired. Should I add a separate commit in this PR, a new PR, not change them?